content/english/pygame-pong/the-basics.md

```diff
@@ -0,0 +1,82 @@
+---
+title: "The Basics - Class and Object"
+date: 2019-07-29T13:24:17-07:00
+weight: 2
+draft: false
+
+---
+## What are class and object ? 
+Before you start building a game, you should know how to draw different objects and modify them as per the way they need to look and move on computer screen. 
+
+In order to build objects you first decide how they should look, what role they would perform in your game and how player can control their movement on screen. You do this by creating a class for each object you want on screen. 
+Before objects are created in a program you must define their class. A class is what tells how an individual object would look and behave. It contains the attributes and methods that its object can use during the lifetime of a program (in this case it is Pong game). You need to define class once and you can create as many objects you wish from that class. 
+
+You cannot use class directly in a program, a class must have an object as its manifestation that is used in program. 
```

Comment:
This might be a bit hard to digest. For now, let's leave it as is - we will need to move this to the "basics" curriculum as well.

Reply:
So should I remove from file or leave as it is checked in now ?